Раджабов Роман

RadzhabovRoman opened the pull request:
Гитхаб живи
[Посмотреть решение](https://hruview.herokuapp.com/RadzhabovRoman/markup-task-3/master/index.html)

RadzhabovRoman committed:
Решение

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

RadzhabovRoman committed:
Подправил раз

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

RadzhabovRoman committed:
Гитхаб живет

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

RadzhabovRoman committed:
Картинки уехали - починил

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

spt30 commented on index.css:
```diff
@@ -0,0 +1,156 @@
+    column-gap: 15px;
+}
+
+.columnBOX
```
Интересный метод наименования классов

spt30 commented on index.css:
```diff
@@ -0,0 +1,156 @@
+    position: relative;
+    text-align: justify;
+    margin-left: 31%;
+    width: 69%;
```
что-то не влезло:(

![image](https://user-images.githubusercontent.com/31798669/48720969-b98fb600-ec42-11e8-8da3-d1d5d2229238.png)

spt30 commented on index.css:
```diff
@@ -0,0 +1,156 @@
+{
+    margin-bottom: 50px;
+    margin-top: 50px;
+
```
строчка лишняя

spt30 commented on index.html:
```diff
@@ -3,7 +3,361 @@
+        src="https://cdn1.savepice.ru/uploads/2018/10/21/8360b0fb18d4467149256d3cfcd93ab0-full.png"
+        alt="Британский спорт"
+    >
+    <h1>СПОРТИВНЫЕ ИГРЫ В ШКОЛАХ</h1>
```
Хотелось бы, чтобы тут был обычный текст, а регистр для оформления задан с помощью стилей

spt30 replied:
Это не поправлено, но раз ты не поправил, то можешь в будущем сам разобраться, как это делать

spt30 commented on index.html:
```diff
@@ -3,7 +3,361 @@
+            </p>
+            <p class="star">☆☆☆☆☆☆</p>
+            <p class="basic">
+                <span class="firstLetterIntro">Т</span>а&shy;кое по&shy;ло&shy;же&shy;ние
```
Есть специальный псевдоэлемент для первой буквы, используй его

spt30 commented on index.html:
```diff
@@ -3,7 +3,361 @@
+    <section>
+        <aside class="box">
+            <div>
+                <p>Б</p>
```
Есть способ это тоже сделать с помощью css

spt30 reviewed (changes requested):
🍅 Привет! Ты сдал задачу за 15 часов до дедлайна, новый до 27 ноября 13:00

Пробегись, пожалуйста, еще раз по условию: все пункты обязательные
И переименуй классы в другом стиле, пожалуйста)

RadzhabovRoman committed:
Учёл замечания

1)Добавил картинку наперерез 2-х колонок (если правильно вас понял, то этого не хватало)
2)Переименовал классы
3)Убрал ненужные классы и строчки, использовав firs-letter
4)Картинка снизу теперь влезает при любом масштабировании
5)БОКС выровнял так, потому что text-orientation не работает в IE

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

spt30 commented:
🍏 основные моменты поправлены, молодец)
